src/Raven.Studio/typescript/models/database/tasks/ongoingTaskOlapEtlTransformationModel.ts

```diff
@@ -0,0 +1,91 @@
+﻿/// <reference path="../../../../typings/tsd.d.ts"/>
+import collectionsTracker = require("common/helpers/database/collectionsTracker");
+import jsonUtil = require("common/jsonUtil");
+
+class ongoingTaskOlapEtlTransformationModel {
+    name = ko.observable<string>();
+    script = ko.observable<string>();
+    collection = ko.observable<string>();    
+    
+    collectionColorIndex: KnockoutComputed<number>;    
+    isNew = ko.observable<boolean>(true);
+    resetScript = ko.observable<boolean>(false);
+    
+    validationGroup: KnockoutValidationGroup; 
+  
+    dirtyFlag: () => DirtyFlag;
+    
+    constructor(dto: Raven.Client.Documents.Operations.ETL.Transformation, isNew: boolean, resetScript: boolean) {
+        this.update(dto, isNew, resetScript);
+        
+        this.initObservables();
+        this.initValidation();
+    }
+
+    initObservables() {
+        this.collectionColorIndex = ko.pureComputed(() => collectionsTracker.default.getCollectionColorIndex(this.collection()));
+        
+        this.dirtyFlag = new ko.DirtyFlag([
+            this.name, 
+            this.script,
+            this.collection,
+            this.resetScript
+        ], false, jsonUtil.newLineNormalizingHashFunction);
+    }
+   
+    static empty(): ongoingTaskOlapEtlTransformationModel {
+        return new ongoingTaskOlapEtlTransformationModel(
+            {
+                ApplyToAllDocuments: false, 
+                Collections: [],
+                Disabled: false,
+                Name: "",
+                Script: ""
+            }, true, false);
+    }
+
+    toDto(): Raven.Client.Documents.Operations.ETL.Transformation {
+        return {
+            ApplyToAllDocuments: false,
+            Collections: [this.collection()],
```

Comment:
why we support single collection? see: https://github.com/ravendb/ravendb/blob/c329d57f62a480d9b66e0bcc5c110b286d676c3b/src/Raven.Studio/typescript/models/database/tasks/ongoingTaskRavenEtlTransformationModel.ts#L59

Reply:
done, now support multiple